apps/condo/domains/common/components/Import/Index.tsx

```diff
@@ -86,6 +87,7 @@ export const ImportWrapper: React.FC<IImportWrapperProps> = (props) => {
         objectsName,
         accessCheck,
         columns,
+        maxTableLength,
```

Comment:
С учетом того, что этот параметр может быть опциональным
```suggestion
        maxTableLength = 500,
```